### DISCUSSION TITLE
How to return an error message to the client via `input_fn`

### DISCUSSION THREAD

**Discussion by dshahrokhian** [aws/sagemaker-python-sdk#2380] [Help]
**What did you find confusing? Please describe.**
In none of the examples I've read, I have found an example of returning an error to the client, when the requested format is not supported.

I have tried raising `ValueError` within the method, but then the client just receives error 500, not the reason.

**Describe how documentation can be improved**
Including in [readthedocs](https://sagemaker.readthedocs.io/en/stable/) (for example, [this documentat](https://sagemaker.readthedocs.io/en/stable/using_sklearn.html#process-input)) an example on how we can return a particular error message to the client.

**Comment by laurenyu** [comment]
@dshahrokhian sorry for the delay in response here. what framework and framework version are you using? (the answer varies based on that)
